homeassistant/components/netatmo/data_handler.py

```diff
@@ -365,9 +390,19 @@ def setup_modules(self, home: pyatmo.Home, signal_home: str) -> None:
         }
         for module in home.modules.values():
             if not module.device_category:
+                _LOGGER.debug(
```

Comment:
I would suggest moving these new logs out of this PR and into a separate one, as recommended in https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr